utils/ccformat.py

```diff
@@ -43,6 +43,8 @@ def runTidy(args):
                            "LLVM_BUILD environment variable."
       return 1
 
+    coreclrbuild = expandPath(args.coreclr_build)
```

Comment:
This is a path to the CORECLR build directory CoreClr\bin\Product\Windows_NT.x64.debug